KomaMRIBase/src/datatypes/Sequence.jl

```diff
@@ -296,6 +296,61 @@ The total duration of the sequence in [s].
 """
 dur(x::Sequence) = sum(durs(x))
 
+"""
+    T0 = get_block_start_times(seq::Sequence)
+
+Returns a vector containing the start times of blocks in a sequence. The initial time is
+always zero, and the final time corresponds to the duration of the sequence.
+
+# Arguments
+- `seq`: (`::Sequence`) Sequence struct
+
+# Returns
+- `T0`: (`::Vector`, `[s]`) start times of the blocks in a sequence
+"""
+get_block_start_times(seq::Sequence) = cumsum([0; seq.DUR], dims=1)
+
+"""
+    samples = get_samples(seq::Sequence; off_val=0, max_rf_samples=Inf)
+
+Returns the samples of the events in a Sequence.
```

Comment:
```suggestion
Returns the samples of the events in `seq`.
```

Reply:
Addressed in https://github.com/cncastillo/KomaMRI.jl/pull/240/commits/a7bff9354c965bc1cd07216ff427da9d84ee75ee